.github/workflows/teleport-create-user.yaml

```diff
@@ -0,0 +1,58 @@
+name: Create teleport user
+on:
+  workflow_call:
+      inputs:
+        cluster_teleport:
+          description: "Address teleport cluster"
+          type: string
+          default: "teleport.dev.aml-ai.net"
+          required: true
+        OTP-admin:
+          description: "Input OTP admin"
+          type: string
+          default: "123456"
+          required: true
+        user_name:
+          description: "Create user namme"
+          type: string
+          required: true
+        user_role:
+          description: "Assign roles to user"
+          type: multi-choice
+          options:
```

Comment:
I forgot to mention you that it was decided that the roles have to be three, as you see here: https://hawkai.atlassian.net/wiki/spaces/SRE/pages/3705438215/Remote+Endpoints+Access+Matrix

Reply:
In the test I made, I created the developer user and the super-admin exactly as you did, and for the middle one, "admin", I gave him access to 2 labels, developer AND admin. So that applications with the "admin" label can be seen by admins only (not developers) but admins are able to see applications meant both for developers and admins. But you are free to find a better solution, I didn't think much about that ; )